cx_plugins/executive_plugin/clips/time.clp

```diff
@@ -12,6 +12,7 @@
 ; See the License for the specific language governing permissions and
 ; limitations under the License.
 
+(defmodule MAIN (export deftemplate time))
```

Comment:
Why is this necessary?

Shouldn't this be up to the user (e.g., to re-define the main module once and then exporting whatever is necessary)?

I have not worked with modules yet, but it seems if this export here is necessary, then this needs to be a policy across all plugins.